test/common/link-patterns.json

```diff
@@ -25,5 +25,15 @@
         "label": "coverage",
         "pattern": "Code coverage report in ([A-Za-z0-9\\-\\.]+)$",
         "url": "$1/"
+    },
+    {
+        "label": "vm xml",
+        "pattern": "Wrote ([A-Za-z0-9\\.\\-]+) XML to ([A-Za-z0-9\\.\\-]+\.xml)$",
```

Comment:
use a raw string here; the other patterns ought to do that as well, to avoid the ugly double `\\`.